README.md

```diff
@@ -36,78 +39,75 @@ npm i @liftric/auth@<version>
 
 ## How-to
 
-### Instantiating
-
-The handler needs a configuration object consisting of the region code and the client ID.
-
-```kotlin
-val configuration = Configuration(region = Region.euCentral1,
-                                  clientId = "CLIENT_ID") 
-```
+### Init
 
 #### Kotlin
 
 ```kotlin
-val authHandler = AuthHandler(configuration) 
+val provider = IdentityProvider(Region.EUCentral1, "<clientId>") 
```

Comment:
I do like this approach much more than having an additional configuration object!